test/olm/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM quay.io/openshift/origin-operator-registry:latest
+
+# The operator image location should be updated before copying.
```

Comment:
I tried to replace pass image location as an argument (`ARG image`) and replace it within the container using `sed`. But I am getting a permission error. If we can fix it, that would be a better approach. Let me know if anyone has got a pointer?

The current approach that I plan to take up is to replace it in the original location and revert it later.

Reply:
Created an issue to track it for now: #32